.eslintignore

```diff
@@ -1,6 +1,5 @@
 coverage/
 dist-*/
 docs/
-browsertest/browsertest.js
-browsertest/browsertest.min.js
+browsertest/
```

Comment:
This excludes setup files etc.?

Reply:
Yes, but do we need it for 2 lines of code? The current setup did not let me pass CI